src/helpers/visibilityClassnames.ts

```diff
@@ -0,0 +1,38 @@
+import { display } from 'classnames/tailwind'
+
+type Size = 'xs' | 'sm' | 'md' | 'md700' | 'lg'
```

Comment:
I don't think we need `'md700'` because it's used only on SealCred for now.

Reply:
not actually, since SealVerse will be on every Seal* website, we need it on SealCaster and we'll need it to SCE also